src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketCommit.java

```diff
@@ -48,7 +48,7 @@
     String getDate();
 
     /**
-     * @return the commit hash (complete, not reduced)
+     * @return the commit hash (short one for Bitbucket Cloud)
```

Comment:
During my tests, I noticed that Bitbucket Cloud REST API is returning a short commit sha1, not the full one. But this doesn't prevent to call other APIs using this short commit ref (for example I tested creating a build status).

Reply:
Yes, the REST API returns the short format, but then the [API client resolves the long one](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/7c761939e6b950adbadb08ede737ac3b51470002/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java#L471) so `BitbucketCommit` instances always hold the long format.